CHANGELOG_PENDING.md

```diff
@@ -1,4 +1,4 @@
-## v0.27.4
+## v0.28.0
```

Comment:
Just looks like we forgot to bump it when we released v0.27.4. Don't think it's a big deal or needs it's own peer since it's not a full version bump / release prep